docs/src/main/sphinx/connector/hive.rst

```diff
@@ -388,6 +388,12 @@ Property Name                                      Description
 ``hive.query-partition-filter-required``           Set to ``true`` to force a query to use a partition filter.   ``false``
                                                    You can use the ``query_partition_filter_required`` catalog
                                                    session property for temporary, catalog specific use.
+
+``hive.partition_statistics_sample_size``          You can use the ``hive.partition_statistics_sample_size``     100
+                                                   catalog session property to view a sample size.
```

Comment:
Is it used to set the set size of provided samples, or does it set the sample size? If you "view the sample size" the value viewed should be "10" or whatever the sample size is set to.

What I think this is doing is setting the size of a data sample.